Upstream: https://github.com/vslavik/diff-pdf/pull/69
Reason: Poppler Cairo was removed from Poppler and is not required anymore

diff --git a/configure.ac b/configure.ac
index 382e220..2e9b6e5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -41,7 +41,7 @@ AC_LANG(C++)
 dnl === Library checks ===
 
 PKG_CHECK_MODULES(POPPLER,
-                  [poppler-cairo >= 0.10 poppler-glib >= 0.10 cairo-pdf])
+                  [poppler-glib >= 0.10 cairo-pdf])
 
 AM_OPTIONS_WXCONFIG
 AM_PATH_WXCONFIG([3.0.0], [wxfound=1], [wxfound=0], [core,base])
